modules/kernel/src/main/java/org/opencastproject/kernel/security/OrganizationDirectoryServiceImpl.java

```diff
@@ -192,14 +190,28 @@ public void updated(String pid, Dictionary properties) throws ConfigurationExcep
 
     // Build the properties map
     final Map<String, String> orgProperties = new HashMap<String, String>();
+    ArrayList<String> serverUrls = new ArrayList();
+
     for (Enumeration<?> e = properties.keys(); e.hasMoreElements();) {
       final String key = (String) e.nextElement();
+
       if (!key.startsWith(ORG_PROPERTY_PREFIX)) {
         continue;
       }
+
+      if (key.startsWith(ORG_SERVER_KEY)) {
+        String tenantSpecificHost = StringUtils.trimToNull((String) properties.get(key));
+        serverUrls.add(tenantSpecificHost);
+      }
+
       orgProperties.put(key.substring(ORG_PROPERTY_PREFIX.length()), (String) properties.get(key));
     }
 
+    if (serverUrls.isEmpty()) {
+      logger.debug("No server configured for organization " + name + ", setting default");
+      serverUrls.add(DEFAULT_SERVER);
```

Comment:
Just wondering what happens if no server is set?

Reply:
We fall back to localhost, which is what happened before (just in a different way). And there won't be any mapping, since it's not configured.

Reply:
I meant, do we need to set something at all?

Reply:
I traced it back to the point where it's used, and it's falling back to the default organization there anyway, so I'm gonna remove this.